lib/runners/processor/actionlint.rb

```diff
@@ -0,0 +1,83 @@
+module Runners
+  class Processor::Actionlint < Processor
+    SCHEMA = _ = StrongJSON.new do
+      extend Schema::ConfigTypes
+
+      # @type self: SchemaClass
+      let :config, base(
+        target: target,
+
+        # Native options
+        ignore: one_or_more_strings?,
+      )
+    end
+
+    register_config_schema SCHEMA.config
+
+    def self.config_example
+      <<~'YAML'
+        root_dir: project/
+        target:
+          - .github/workflows/one_worflow.yml
+          - .github/workflows/another_worflow.yml
+        ignore:
+          - "job .* needs job"
+          - "unknown Webhook event"
+      YAML
+    end
+
+    # This analyser uses git metadata (.git/).
+    def use_git_metadata_dir?
+      true
+    end
+
+    def analyzer_options
+      opts = []
+      opts << "-format"
+      opts << "{{json .}}"
+
+      Array(config_linter[:ignore]).each do |rule|
+        opts << "-ignore"
+        opts << rule
+      end
+
+      if config_linter.key?(:target)
+        opts << "--"
+        opts += Array(config_linter[:target])
+      end
+      opts
+    end
+
+    def analyze(changes)
+      stdout, stderr, status = capture3(
+        analyzer_bin,
+        *analyzer_options,
+      )
+
+      unless [0, 1].include?(status.exitstatus)
+        failure_message = ("actionlint failed with: " + stderr).rstrip
```

Comment:
[suggestion] You can use the `#analyzer_name` method. 😉 

```suggestion
        failure_message = ("#{analyzer_name} failed with: " + stderr).rstrip
```

Reply:
Thank you. Changed.